src/tests/icusd.test.ts

```diff
@@ -18,11 +22,20 @@ describe('icUSD (mainnet)', () => {
   })
 
   test('can mint with USDC', async () => {
+    const usdcBalance = await getUsdcBalance(
+      usdc.address as Address,
+      indexToken.address as Address,
+      1
+    )
+    // Minting enough tokens for the redemption tests
+    const inputAmountGreaterThreshold = (usdcBalance * BigInt(90)) / BigInt(100)
     const quote = await factory.fetchQuote({
       isMinting: true,
       inputToken: usdc,
       outputToken: indexToken,
-      indexTokenAmount: wei('1'),
+      // Index token amount will be ignored for minting
+      indexTokenAmount: wei(formatUnits(inputAmountGreaterThreshold, 6)),
+      inputTokenAmount: BigNumber.from(inputAmountGreaterThreshold.toString()),
```

Comment:
Note that depending on if FMWrapped or FMNav is used you'll either have to set `inputTokenAmount` or `indexTokenAmount`. Final goal here is that minting should use FMNav, so for testing you only need to change `inputTokenAmount`. But if you wanna test with FMWrapped (see temp change further above in routing code), you need to set `indexTokenAmount`